spk/ffmpeg/Makefile

```diff
@@ -12,6 +12,13 @@ DISPLAY_NAME = ffmpeg
 STARTABLE = no
 
 HOMEPAGE = http://www.ffmpeg.org/
-LICENSE  = LGPL 2.1
+LICENSE  = GPLv3
+
+PRE_COPY_TARGET = ffmpeg_custom_pre_copy
 
 include ../../mk/spksrc.spk.mk
+
+.PHONY: ffmpeg_custom_pre_copy
+ffmpeg_custom_pre_copy: $(INSTALL_PLIST)
+	# Remove libhdr10plus reference from PLIST if it isn't available on the platform (e.g. 88f6281)
+	test -f $(INSTALL_DIR)$(INSTALL_PREFIX)/lib/libhdr10plus.so || sed -i -e '/:lib\/libhdr10plus.so/d' $(INSTALL_PLIST)
```

Comment:
@m4tt075 Such PLIST operation already has a support in framework... I have discovered thanks to Allan commit with PLIST_TRANSFORM variable at https://github.com/SynoCommunity/spksrc/pull/3562/commits/a037093b9c58a39e46528fbbedadcc5121d81066

Reply:
@ymartin59 I'm struggeling to implement your request. According to my testing, **on shell level**, the following command (all in one row) should work within the pipe:
`PLIST_TRANSFORM = if [ -f $(INSTALL_DIR)$(INSTALL_PREFIX)/lib/libhdr10plus.so ] ; then sed -e '/lib\/libhdr10plus.so/d' ; else cat ; fi` 
However, if I insert this expression into the Makefile variable, it doesn't work. It might be an escaping issue, but all of my attempts failed so far. Can you advise, please?